wiki/en/en-Sound-Devices.md

```diff
@@ -1,87 +1,138 @@
 ---
 layout: wiki
-title: "Known Good Devices"
+title: "Sound Devices"
 lang: "en"
 permalink: "/wiki/Sound-Devices"
 ---
 
-# Known Good Devices
+# Sound Devices
 
-This page lists hardware known to work or not work with Jamulus (although it may vary between Windows, Mac and Linux). Please update with any new information you have (and indicate the platform(s) you have used).
+This page lists (thanks to Jamulus users) the audio devices known to work (or not) with Jamulus.  
+_**Note**: depends on how the device works on your system (Linux, MacOS, Windows, etc.)_.
```

Comment:
depends on how --> This depends on ...

Reply:
Also two full stops look strange (end of sentence)

Reply:
Fixed